Tools/BatchScripts/batch_perlmutter.sh

```diff
@@ -10,6 +10,8 @@
 #SBATCH -N 4
 #SBATCH -J WarpX
 #SBATCH -A <proj>
+# for m3906 LBNL/AMP users: for large runs, comment in
+#S BATCH -q early_science
```

Comment:
Is this a typo?
```suggestion
#SBATCH -q early_science
```

Reply:
No, intentionally commented out (see line above).

Not everyone has access to `-q early_science`

Reply:
To me, that reads as a possible type instead of a comment. I'd do double (or more) hashes: `##SBATCH`.

Reply:
Cool, yes that works if double-hashes are commented out.
Cool!
```suggestion
##SBATCH -q early_science
```

Ref.: https://hpc.nmsu.edu/discovery/slurm/slurm-commands/